src/seedu/addressbook/data/Tagging.java

```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.data;
+
+
+import java.util.ArrayList;
+
+public class Tagging {
```

Comment:
Missing header comment. All non-trivial classes and methods should have javadoc format header comments.